packages/theme-chalk/src/tabs.scss

```diff
@@ -392,11 +401,15 @@
 
     .#{$namespace}-tabs__nav.is-left,
     .#{$namespace}-tabs__nav.is-right {
-      float: none;
+      flex-direction: column;
+    }
+
+    .#{$namespace}-tabs__item.is-left {
+      justify-content: flex-end;
     }
-    .#{$namespace}-tabs__item.is-left,
+
     .#{$namespace}-tabs__item.is-right {
-      display: block;
+      justify-content: flex-start;
```

Comment:
Are there compatibility issues with `justify-content`?

Reply:
Never mind. We already use these properties in other compoents.
![image](https://user-images.githubusercontent.com/38392315/226084085-b1a6809c-bc9c-4bc3-b503-0363e54b3fad.png)

Reply:
ok.
/cc @btea

Reply:
Ok.